project.py

```diff
@@ -46,8 +53,19 @@ def __init__(self, n_slides=0,
         # ratio of similarity between images based on which we decide if we
         # are going to save an image
         self.similarity = similarity
+        self.__func_keyword_to_function__(compare_function)
         self.load_config()
 
+    def __func_keyword_to_function__(self, keyword):
+        switcher = {
+            'dhash': '__compute_dhash__',
+            'phash': '__compute_phash__',
+            'ahash': '__compute_ahash__',
+            'orb': None
```

Comment:
@Foxiy I do not quite understand why `'orb'` would be any different than all these other functions. Later on your check whether `self.hash_function` is `None` and if it is, you call `__check_change_orb__` in the same way you call `__compute_dhash__`. I am sure there is some but I fail to see the reason for this.